pkg/cmd/server/api/types.go

```diff
@@ -1328,8 +1328,8 @@ type LDAPQuery struct {
 }
 
 type AdmissionPluginConfig struct {
-	// Location is the path to a configuration file that contains the plugin's
-	// configuration
+	// Location is the path to a legacy configuration file that contains the plugin's
+	// configuration. DEPRECATED.
```

Comment:
At least legacy in kube: https://github.com/openshift/origin/blob/9949be492fe910aa1778032e445ad9b197b60e7a/vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/admission/config.go#L108. Probably, here in the origin config neither "legacy" nor "DEPRECATED" makes much sense. Will remove it.